sdk/javascript/src/impl/ed25519.js

```diff
@@ -0,0 +1,27 @@
+const {
+	crypto_sign_keypair, crypto_private_sign, crypto_private_verify, HashMode
+} = require('../../wasm/pkg/symbol_crypto_wasm');
+
+const CRYPTO_SIGN_BYTES = 64;
+const CRYPTO_SIGN_PUBLICKEYBYTES = 32;
+const CRYPTO_SIGN_SECRETKEYBYTES = 64;
+
+const ed25519 = {
+	keyPairFromSeed: (hashMode, seed) => {
+		const publicKey = new Uint8Array(CRYPTO_SIGN_PUBLICKEYBYTES);
+		const privateKey = new Uint8Array(CRYPTO_SIGN_SECRETKEYBYTES);
+		for (let i = 0; i < CRYPTO_SIGN_SECRETKEYBYTES; i++)
+			privateKey[i] = seed[i];
```

Comment:
I'm not sure if we need privateKey here, I think that:
 * we already require `seed` to be uint8Array
 * I guess we can throw seed into returned dict below (or is there some reason _not_ to?)
  * crypto_sign_keypair does copying to internal sk anyway